src/main/java/at/ac/tuwien/kr/alpha/common/depgraph/ComponentGraph.java

```diff
@@ -0,0 +1,158 @@
+package at.ac.tuwien.kr.alpha.common.depgraph;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+
+import org.apache.commons.lang3.StringUtils;
+
+import at.ac.tuwien.kr.alpha.common.depgraph.StronglyConnectedComponentsHelper.SCCResult;
+
+public final class ComponentGraph {
```

Comment:
Please provide a bit of JavaDoc in this file.

Reply:
done